followthemoney/model.py

```diff
@@ -11,11 +13,11 @@
 class Model(object):
     """A collection of schemata."""
 
-    def __init__(self, path):
-        self.path = path
-        self.schemata = {}
-        self.properties = set()
-        self.qnames = {}
+    def __init__(self, path: str):
+        self.path: str = path
+        self.schemata: Dict[str, Schema] = {}
+        self.properties: Dict[str, Property] = {}
+        self.qnames: Dict[str, str] = {}
```

Comment:
is qnames str not property?

Reply:
woops, good catch... I changed the type of `self.properties` and `generate` is broken right now.